docs/resources.md

```diff
@@ -16,15 +16,16 @@
 
 Our CI handles updating these two files automatically. PRs containing manual updates can often lead to unwanted merge conflicts. Similarly, avoid activating Github actions on your fork so CI doesn't make this problem worse for you.
 
-### Symlink project files to your Community directory
-You can create a symlink (or "directory junction" in Windows terms) between your local repo and the game's community folder. This lets you develop from your local repository without having to copy files over when you're ready to test. Developers working on hot-reloadable files can get changes reflected in real-time, while still retaining a proper development environment
+### Symlink project files to your simulator
+
+You can create a symlink (or "directory junction" in Windows terms) between your local repo and the game's community folder. This lets you develop from your local repository without having to copy files over when you're ready to test. Developers working on hot-reloadable files can get changes reflected in real-time, while still retaining a proper development environment.
 
 First, be sure to remove any existing install of the mod from the game.
 
 Then in `cmd`, run the command below. Be sure to replace each path with the proper respective file path.
 
 ```cmd
-mklink /J "C:\users\<user>\AppData\Microsoft Flight Simulator\Packages\Community\flybywire-aircraft-a320-neo" "C:\path\to\cloned\repo\flybywire-aircraft-a320-neo"
+mklink /J "C:\Users\<user>\AppData\Roaming\Microsoft Flight Simulator\Packages\Community\flybywire-aircraft-a320-neo" "C:\path\to\cloned\repo\flybywire-aircraft-a320-neo"
```

Comment:
Path is `\AppData\Local\` not `\AppData\Roaming\`

Reply:
It is under `Roaming` for my installation - maybe because I have installed it using Steam?

Reply:
Yeah thats why, maybe you could specify for each edition?

Reply:
```\AppData\Roaming\``` is the path for Steam. Can someone confirm for Store edition

Reply:
I have local listed on our docs site -> [here](https://docs.flybywiresim.com/start/installation/#microsoft-store-andor-game-pass-edition) for Store edition. This was also after an update to both our repo README.md and the docs site. Should be valid.